tests/integration/ha_tests/test_async_replication.py

```diff
@@ -434,7 +434,6 @@ async def test_async_replication_failover_in_main_cluster(
     # Check that the sync-standby unit is not the same as before.
     new_sync_standby = await get_sync_standby(ops_test, first_model, DATABASE_APP_NAME)
     logger.info(f"New sync-standby: {new_sync_standby}")
-    assert new_sync_standby != sync_standby, "Sync-standby is the same as before"
```

Comment:
I requested this change as Patroni is recovering quickly and becoming the `sync-standby` again in the former unit. We shouldn't always expect the behaviour tested on this check.

Reply:
The way I see it, this specific change may be a little out of scope for this rolling-ops update PR, I propose to revert this change and create a follow-up ticket to investigate the underlying issue at play here. What do you think?

Reply:
change reverted and follow-up ticket created https://warthogs.atlassian.net/browse/DPE-4550